kaspresso/src/main/kotlin/com/kaspersky/kaspresso/autoscroll/ObjectAutoScrollProviderImpl.kt

```diff
@@ -47,32 +47,16 @@ class ObjectAutoScrollProviderImpl(
      * @return the result of [action] invocation.
      */
     override fun <T> scroll(interaction: UiObjectInteraction, action: () -> T, cachedError: Throwable): T {
-        /**
-         *  Looks for a scrollable content
-         */
-        val scrollable = UiScrollable(UiSelector().scrollable(true))
-
-        /**
-         * Scrolls to the bottom and looks for the given view. Invokes the action if the view was found.
-         */
-        do {
-            if (interaction.tryToFindUiObject()) {
-                logger.i("UiView autoscroll successfully performed.")
-                return action.invoke()
-            }
-        } while (scrollable.scrollForward())
-
-        /**
```

Comment:
Removed scrolling up. I think that such behavior is a little bit tricky and unpredictable. That's why the only scrolling bottom remains.

Reply:
It is a bad idea to change the scroll behavior. You can break users' tests

Reply:
Moreover, it could be that the user starts the screen in the middle of the list for whatever reason, like a highlighted element or something similar. I believe it is better to keep it as well

Reply:
Agree. I just remembered a case when up scrolling caused the browser page reloading that was inappropriate. But, yep, you are right.